Philips.CodeAnalysis.MaintainabilityAnalyzers/Naming/EnforceBoolNamingConventionAnalyzer.cs

```diff
@@ -28,7 +28,7 @@ public class EnforceBoolNamingConventionAnalyzer : SingleDiagnosticAnalyzer
 		public EnforceBoolNamingConventionAnalyzer() : this(true, true) { }
 
 		public EnforceBoolNamingConventionAnalyzer(bool shouldCheckLocalVariables, bool shouldCheckFieldVariables)
-			: base(DiagnosticId.EnforceBoolNamingConvention, Title, MessageFormat, Description, Categories.Naming)
+			: base(DiagnosticId.EnforceBoolNamingConvention, Title, MessageFormat, Description, Categories.Naming, isEnabled: false)
```

Comment:
Why disable this Analyzer ?

Reply:
It was already disabled, and I accidentally enabled it in the last refactorings